### PR TITLE
[8.0] Support token transfers in FTS

### PR DESCRIPTION
**Pull request by chaen**
Needs
https://gitlab.cern.ch/fts/fts-rest-flask/-/merge_requests/116

Requires an sql update

```sql
alter table Files CHANGE `status` `status` enum('New','Submitted','Ready','Active','Finished','Canceled','Staging','Failed','Defunct','Started','Not_used','Archiving', 'Token_prep') DEFAULT 'New'
```

BEGINRELEASENOTES

*DMS
NEW: experimental token support for FTS transfers

*Framework

NEW: allow to bypass cache when retrieving tokens

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
So, following the workshop, opinion on having that merged in ?

File: docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst [hunk 1]
```diff
@@ -200,3 +200,27 @@ More details on how the intermediate SE selection is done and how the matrix is
    Work in FTS has a `task <https://its.cern.ch/jira/projects/FTS/issues/FTS-1755>`_ to try and bring that feature in.
    A future solution may come from DIRAC.
    In the meantime, the best solution is to ask the site to either cleanup themselves (some storages like EOS have that built in) or to give you a dump of the namespace, and then do the cleaning yourself.
+
+
+Token support
+----------------
+
+.. versionadded:: v8.0.40
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst]:
```suggestion
.. versionadded:: v8.0.49
```
Or maybe 50?

**Comment by fstagni** [comment]
This requires an SQL update, so it should be a bit more carefully announced.

**Comment by chaen** [comment]
> This requires an SQL update, so it should be a bit more carefully announced.

Sure ! But I'd still like to have other communities (@andresailer @marianne013 @atsareg ) using this system giving their input

**Review by andresailer — COMMENTED** [review]
This is isolated to the rest of the code, so I have no objection for merging it.

**Comment by fstagni** [comment]
Is this finally ready for merge?

**Comment by marianne013** [comment]
I just had a quick chat with Simon and his answer  basically amounted to a shrug, so go ahead (FYI @sfayer)

File: docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst (hunk 1 above)
Comment (fstagni) [reply docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst]: 51

**Review by fstagni — DISMISSED** [review]

**Review by fstagni — APPROVED** [review]

**Pull request merged**